package-lock.json

```diff
@@ -1157,6 +1157,7 @@
         "combined-stream": {
           "version": "1.0.5",
           "bundled": true,
+          "optional": true,
```

Comment:
Why the changes to package-lock.json? These should be ditched for this PR.

Reply:
Whoops- Didn't change those manually, should have been more careful when committing. I removed the changes to package-lock.json